pkgs/development/python-modules/rising/default.nix

```diff
@@ -31,6 +31,13 @@ buildPythonPackage rec {
     hash = "sha256-sBzVTst5Tp2oZZ+Xsg3M7uAMbucL6idlpYwHvib3EaY=";
   };
 
+  # Patch Python 3.12 incompatibilities in versioneer.py.
+  postPatch = ''
+    substituteInPlace versioneer.py \
+      --replace-fail SafeConfigParser ConfigParser \
+      --replace-fail readfp read_file
+  '';
+
```

Comment:
A better fix is to use packaged versioneer, as in `pkgs/development/python-modules/skein/default.nix`

Reply:
Indeed, thanks.
I changed it.